settings.gradle

```diff
@@ -2,15 +2,17 @@ pluginManagement {
     repositories {
         mavenCentral()
         jcenter()
+        gradlePluginPortal()
         maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
         maven { url 'https://dl.bintray.com/kotlin/kotlin-dev' }
         maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
`https://plugins.gradle.org/m2/` seems to be duplicate of `gradlePluginPortal()`, let's leave one of them.
Also I believe, we can remove kotlin-dev repo now.